src/ubuntu-docker/lind-base/etc/tmux.conf

```diff
@@ -0,0 +1,247 @@
+# set -g default-terminal "rxvt-unicode-256color"
+# set -g default-terminal "screen-256color-it"
+set -g default-terminal "tmux-256color-italic"
+# set -g default-terminal "st-256color"
+set -g escape-time 10
+set -g status-right '"#{=22:pane_title}"'
+set -ga terminal-overrides ",xterm-termite:Tc"
+set -ga terminal-overrides ",xterm*:smcup@:rmcup@"
+set -ga terminal-overrides ",rxvt-uni*:smcup@:rmcup@:XT:Ms=\E]52;%p1%s;%p2%s\007"
+set -ga terminal-overrides ",st*:smcup@:rmcup@:Tc"
+set -g history-limit 30000
+setw -g aggressive-resize on
+setw -g xterm-keys on
+
+# Look & Feel
+#-----------------------------------------------
+
+# Statusbar properties
+##set -g display-time 5000
+set -g status-bg default
+set -g status-fg black
+
+# pane divider color; non active pane
+set -g pane-border-fg green
+set -g pane-border-bg default
+
+# pane divider color; active pane
+# helps when you have more than two panes.
+set -g pane-active-border-fg blue
+set -g pane-active-border-bg default
+
+# window notifications; display activity on other window
+set -g monitor-activity on
+set -g visual-activity on
+
+# statusbar colors
+set -g status-fg white
+set -g status-bg default
+set -g status-attr default
+
+# window title colors; non active
+set -g window-status-fg blue
+set -g window-status-bg black
+set -g window-status-attr dim
+
+# window title colors; active
+set -g window-status-current-fg colour38
+set -g window-status-current-bg default
+set -g window-status-current-attr bright
+
+# urlscan as context and url view
+bind u capture-pane \; save-buffer /tmp/tmux-buffer \; \
+new-window -n "urlscan" '$SHELL -c "urlscan  < /tmp/tmux-buffer"'
+
+# urlview as context and url view
+bind U capture-pane \; save-buffer /tmp/tmux-buffer \; \
+new-window -n "urlview" '$SHELL -c "urlview  < /tmp/tmux-buffer"'
+
+# Titles
+set -g set-titles on
+set -g set-titles-string "#T"
+#set -g set-titles-string "#h - #W"
+#set -g set-titles-string "#(tmux ls | awk -F: '{print $1}' | xargs | sed 's/\ / | /g')"
+set -g status-justify centre
+set -g automatic-rename on
+
+#tmux messages
+set -g message-bg default
+set -g message-fg yellow
+
+#statusbar & external scripts
+#-----------------------------------------------------
+##########################################
+# STATUS BAR
+set -g status on
+# Turning the status line off, or changing its colour:
+#set -g status off
+#set -g status-style bg=blue
+#set -g status-utf8 on
+set -g status-interval 1
+set -g status-attr bright
+set -g status-fg white
+set -g status-bg black
+set -g status-left-length 20
+set -g status-left "∙ #[fg=colour240,bg=white] %b %d %Y #[bg=colour240,fg=white] #(tmux-mem-cpu-load) #[default]"
+set -g status-right '#(~/bin/tmux_battery_charge_indicator.sh) #[bg=colour240,fg=white] %I:%M:%S %P #[bg=white,fg=colour240] %a '
+set -g status-justify centre
+set -g window-status-current-format '#[fg=green,bg=black](#I.#P|#F|#W)#[default]'
+set -g window-status-format '#[fg=lightcyan,bg=black]#I.#P#F#W'
```

Comment:
Is this an auto generated file?

Reply:
no this again is an existing tmux config we were using in arch linux